apps/web/src/config/index.ts

```diff
@@ -7,15 +7,16 @@ declare global {
   }
 }
 
+const isCypress = (isBrowser() && (window as any).Cypress) || (isBrowser() && (window as any).parent.Cypress);
```

Comment:
I've used the `cy.origin` a new functionality that allows using Cypress cross domains, but currently there is a bug which clears the `window.Cypress` when navigating back from the origin, that's an additional check `window.parent.Cypress` is needed. 
More here: https://github.com/cypress-io/cypress/issues/25394